### PR TITLE
remove unwanted file and empty space

### PR DESCRIPTION
**Pull request by muneeswari22**
## Release Notes

- **New Features**
	- Introduced a new `PersistentLogins` class for managing persistent login records.
	- Added a new `User` class for user management with enhanced security features.

- **Bug Fixes**
	- Improved error handling in the `login` method of the `AuthController`.

- **Removals**
	- Removed the `NotificationHelper` class and its notification sending functionality.
	- Eliminated the `DashboardController` and the `IsScreenAccess` middleware.
	- Deleted various unused configuration entries and environment variables.
	- Removed the `loadMenu` route from the API.

- **Updates**
	- Simplified link generation in the `MailHelper` class.
	- Streamlined middleware configuration by removing unnecessary aliases.
	- Enhanced readability in various classes by adopting shorthand array syntax.

### CODE REVIEW

File: app/Http/Controllers/CommonController.php [hunk 1]
```diff
@@ -1,55 +1,22 @@
...
-use App\Helper\GeneralHelper;
+
 
 
 class CommonController extends Controller
 {
-
     public function getHeaders(Request $request) {
-
         $resVal = array();
```
Comment (stutzenmaha) [inline app/Http/Controllers/CommonController.php:14]: change [] instead of array()

Comment (stutzenmaha) [reply app/Http/Controllers/CommonController.php:14]: check all place

File: app/Http/Controllers/AuthController.php [hunk 2]
```diff
@@ -70,15 +64,14 @@ public function login(Request $request)
...
-        $userDetails = DB::table('tbl_user as u')
-            ->leftjoin('tbl_role as r','r.id','=','u.role_id')
-            ->select('u.id','u.company_id','fname', 'lname', 'email', 'u.role_id','r.name as role_name', 'username', 'phone', 'otp', 'employee_id','is_app_login','is_web_login', 'user_code',
-            'u.is_active','u.created_by','u.updated_by', 'u.comments','user_code')
-            ->where('username', $userName)->where('u.is_active', 1)->first();
+        $userDetails = DB::table('tbl_user')
+                        ->select('name','username','password','otp','remember_token','trial_at','is_active','created_by','updated_by', 'comments')
+                        ->where('username', $userName)
+                        ->where('is_active', 1)
+                        ->first();
```
[the remark this answers is not in the document]
Comment (stutzenmaha) [reply app/Http/Controllers/AuthController.php:74]: REmove password,otp,remember token

File: app/Http/Controllers/CommonController.php (hunk 1 above)
Comment (muneeswari22) [reply app/Http/Controllers/CommonController.php]: done

File: app/Http/Controllers/AuthController.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (muneeswari22) [reply app/Http/Controllers/AuthController.php:74]: done

**Pull request merged**